src/num_methods/binary_search.md

```diff
@@ -138,6 +138,63 @@ Another noteworthy way to do binary search is, instead of maintaining an active
 
 This paradigm is widely used in tasks around trees, such as finding lowest common ancestor of two vertices or finding an ancestor of a specific vertex that has a certain height. It could also be adapted to e.g. find the $k$-th non-zero element in a Fenwick tree.
 
+## Parallel Binary Search
+
+<small>Note that this section follows the description in [Sports programming in practice](https://kostka.dev/sp/).</small>
```

Comment:
Is the intention here to provide a reference for further reading, or an attribution? I think it is more common to integrate references in the text (see e.g. how [this](https://ai.googleblog.com/2006/06/extra-extra-read-all-about-it-nearly.html) is linked above in the article) or put them in some kind of further reading section at the end.

Also, to make sure, you understand that by putting the text from the book here you also make it licensed under CC BY-SA 4.0?